pom.xml

```diff
@@ -63,6 +57,67 @@
 
 	<profiles>
 		<profile>
+			<id>ossrh</id>
+			<distributionManagement>
```

Comment:
An example from one of my other projects is:

	<distributionManagement>
		<snapshotRepository>
			<id>sonatype-nexus-snapshots</id>
			<name>Sonatype Nexus Snapshots</name>
			<url>https://oss.sonatype.org/content/repositories/snapshots/</url>
		</snapshotRepository>
		<repository>
			<id>sonatype-nexus-staging</id>
			<name>Nexus Release Repository</name>
			<url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
		</repository>
	</distributionManagement>